.shippable.yml

```diff
@@ -79,7 +79,12 @@ build:
     - _make PLATFORM=imx-mx7dsabresd
     - _make PLATFORM=imx-mx7ulpevk
     - _make PLATFORM=imx-imx8mmevk
+    - _make PLATFORM=imx-imx8mnevk
     - _make PLATFORM=imx-imx8mqevk
+    - _make PLATFORM=imx-imx8qxpmek
+    - _make PLATFORM=imx-imx8qxplpddr4arm2
```

Comment:
Please drop arm2 board. We not upstream validation board to community, customer not able to get this board.

Reply:
Oups, I did not pay attention. I will fix it.